services/apps/alcs/src/home/home.controller.ts

```diff
@@ -160,6 +161,8 @@ export class HomeController {
           completedAt: subtask.completedAt?.getTime(),
           paused: false,
           title: `${recon.application.fileNumber} (${recon.application.applicant})`,
+          appType: recon.application.type,
+          parentType: 'reconsideration',
```

Comment:
Is parentType typed? If yes then perfect otherwise I think it is better to have a type: 'reconsideration'|'application'...

Reply:
Good suggestion, added the union type as suggested.